ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1619,26 +1618,29 @@ export const WorkspaceEdit = fp.flow(
                 />
               </WorkspaceEditSection>
             )}
-            {(!this.isMode(WorkspaceEditMode.Edit) ||
-              this.props.workspace.accessLevel ===
```

Comment:
this was an insufficient check: Owner is not enough.  The new logic is below here.

Reply:
moved to `canSetBillingAccount()`